primary.tf

```
@@ -57,13 +57,15 @@ resource "google_compute_instance" "primary" {
 
 data "google_dns_managed_zone" "dnszone" {
   name = "${var.dns_zone}"
+  project = "${var.dns_project == "" ? "${var.project}" : var.dns_project}"
```

do we need the `"${}"` around the var.project in this? or is 
```suggestion
  project = "${var.dns_project == "" ? var.project : var.dns_project}"
```
ok?

---

Danke! You are correct. Fixed now.